db/migrate/20220211062656_add_nickname_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNicknameToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :nickname, :string
+    add_index :users, :nickname, unique: true
```

Comment:
🤔 I don't see any reason why we'd have to make nicknames unique. I think it would be more fun if we leave this as a free for all but only limit to certain characters: upper/lower case alphanumeric and maybe space only. Makes me wonder if we've done some regex validations before.